lib/fetch/fetchRequest.ts

```diff
@@ -52,7 +52,7 @@ function fetchRequest(method: string, url: string, args: FetchOptions) {
   if (contentType === 'application/json' && body && typeof body !== 'string') {
     body = JSON.stringify(body);
   }
-
+  var fetch = global.fetch || crossFetch;
```

Comment:
Can we add a test to make sure global fetch is called if fetch is defined else crossFetch is called.